tests/integration_test/action_service_test.py

```diff
@@ -2174,6 +2299,160 @@ def test_form_validation_action_with_form_slot_type_slot(self):
+        Actions(name='http_action', type=ActionType.http_action.value, bot=bot, user=user).save()
+        Actions(name=action_name, type=ActionType.form_validation_action.value, bot=bot, user=user).save()
+        FormValidationAction(name=action_name, slot=slot, validation_semantic=semantic_expression, bot=bot, user=user,
+                             slot_set_pre_validation=FormSlotSetPreValidation(type=FORM_SLOT_SET_TYPE.action.value, value='http_action')).save()
+        Slots(name=slot, type='text', bot=bot, user=user).save()
+
+        HttpActionConfig(
+            action_name='http_action',
+            response=HttpActionResponse(dispatch=False,
+                                        value="Chennai"),
+            http_url="http://localhost:8081/mock",
+            request_method="GET",
+            headers=[HttpActionRequestBody(key="botid", parameter_type="slot", value="bot", encrypt=True),
+                     HttpActionRequestBody(key="userid", parameter_type="value", value="1011", encrypt=True),
+                     HttpActionRequestBody(key="tag", parameter_type="value", value="from_bot", encrypt=True)],
+            params_list=[HttpActionRequestBody(key="bot", parameter_type="slot", value="bot", encrypt=True),
+                         HttpActionRequestBody(key="user", parameter_type="value", value="1011", encrypt=False),
+                         HttpActionRequestBody(key="tag", parameter_type="value", value="from_bot", encrypt=True)],
+            bot=bot,
+            user=user
+        ).save()
+
+        http_url = 'http://localhost:8081/mock'
+        resp_msg = "Chennai"
+        responses.reset()
+        responses.start()
+        responses.add(
+            method=responses.GET,
+            url=http_url,
+            json=resp_msg,
+            status=200,
+            match=[responses.matchers.json_params_matcher(
+                {"bot": "5f50fd0a56b698ca10d35d2e", "user": "1011", "tag": "from_bot"})],
+        )
+
+        responses.add(
+            method=responses.POST,
+            url=Utility.environment['evaluator']['url'],
+            json={"success": True, "data": True},
+            status=200,
+            match=[responses.matchers.json_params_matcher(
+                {'script': semantic_expression,
+                 'data': {'sender_id': 'default', 'user_message': 'get intents',
+                          'slot': {'bot': '5f50fd0a56b698ca10d35d2e', 'current_location': 'Chennai',
+                                   'requested_slot': 'current_location'}, 'intent': 'test_run', 'chat_log': [],
+                          'key_vault': {},
+                          'kairon_user_msg': None, 'session_started': None}}
+            )],
+        )
+        request_object = {
+            "next_action": action_name,
+            "tracker": {
+                "sender_id": "default",
+                "conversation_id": "default",
+                "slots": {"bot": bot, slot: 'Chennai', 'requested_slot': slot},
```

Comment:
current value for slot is chennai and response from http action is also chennai. 
Tomorrow if due to regression, the http action fails, this test case will still pass instead of failing, isnt it?
Unit/integration tests should act as alarm against regression. Write them religiously.